lib/reek/core/examiner.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../source/source_repository'
+require_relative 'warning_collector'
+require_relative 'smell_repository'
+require_relative 'tree_walker'
+
+module Reek
+  module Core
```

Comment:
Now Examiner is scoped under Core so everything is indented two to the right, hence the diff. Same goes for SmellWarning below.